lib/internal/test_runner/runner.js

```diff
@@ -377,20 +386,36 @@ function run(options) {
   if (setup != null) {
     validateFunction(setup, 'options.setup');
   }
+  if (testNamePatterns != null) {
+    if (!ArrayIsArray(testNamePatterns)) {
+      testNamePatterns = [testNamePatterns];
+    }
```

Comment:
is this faster than 
```suggestion
    testNamePatterns = ArrayPrototypeConcat(testNamePatterns);
```
?

Reply:
I honestly don't know 🤔 
Is `concat` quicker? I am happy to change this to concat, but I do I see 2 other times this is implemented: https://github.com/nodejs/node/blob/c7c46e2a523016f93d84a096f9bd51ba6bf6fc0a/lib/_http_outgoing.js#L698-L700 https://github.com/nodejs/node/blob/c7c46e2a523016f93d84a096f9bd51ba6bf6fc0a/lib/internal/errors.js#L1211-L1213

Reply:
Personally I find concat much more idiomatic, but I see a note that it's not a safe primordial to use, so what you have is the better approach :-) please resolve this.